docs/consume-events.md

```diff
@@ -1,35 +1,36 @@
 ---
-title: Consume Events
+title: Consuming Events
 ---
-# What’s a Consumer?
+# What’s a consumer?
```

Comment:
Add `# Consuming Events` at the top and change "What's a consumer?" to an h2: `## What's a consumer?`